app/main.py

```diff
@@ -1 +1,49 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
```

Comment:
You have an extra `else` statement here. According to the checklist, it's recommended to remove the `else` statement if it's not necessary. You can simplify your code by removing the `else` statement and just returning 0 at the end of the function. This works because if `self.is_hungry` is True, the function will return before it gets to the `return 0` statement.